onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -703,26 +702,26 @@ TEST_P(ModelTest, Run) {
           OrtTensorRTProviderOptionsV2 params{0, 0, nullptr, 1000, 1, 1 << 30,
                                               1,  // enable fp16
                                               0, nullptr, 0, 0, 0, 0, 0, nullptr, 0, nullptr, 0, 0, 0};
-          ASSERT_ORT_STATUS_OK(OrtApis::SessionOptionsAppendExecutionProvider_TensorRT_V2(ortso, &params));
+          ortso.AppendExecutionProvider_TensorRT_V2(params);
```

Comment:
Should ASSERT_ORT_STATUS_OK also be changed to release the status object if it's not nullptr? 

Something like https://github.com/microsoft/onnxruntime/blob/0c6037b5abe571fc43a55ef7a9d2f846820fbe5d/onnxruntime/test/shared_lib/fns_candy_style_transfer.c#L11-L20

Reply:
yeah I think that's a good idea. may defer that change to a later PR. trying to get this PR merged soon for the release.